capstone/src/main/webapp/script.js

```diff
@@ -50,4 +50,23 @@ function createClubElement(text) {
   liElement.innerText = text + '  ';
   liElement.innerHTML += '<button>Leave</button>';
   return liElement;
+}
+
+function getClubInfo() {
+  fetch('/clubs').then(response => response.json()).then((clubInfo) => {
+    document.getElementById('club-name').innerHTML = '<h1>' + clubInfo['name'] + '</h1>';
```

Comment:
I think if you change the div tag to an h1 tag on line 12 of about-us.html then you don't need to add the h1 tag here. If this works, then you should be able to do it with the 'description', 'members', and 'website' IDs below, just with p tags instead of h1.